status/projects.py

```diff
@@ -741,21 +741,23 @@ def make_project_running_note(application, project, note, category, user, email)
         for row in v[project]:
             doc_id=row.value
         doc=application.projects_db.get(doc_id)
+        project_name = doc['project_name']
+        proj_ids = [project, project_name]
         doc['details']['running_notes']=json.dumps(running_notes)
         application.projects_db.save(doc)
         #### Check and send mail to tagged users
         pattern = re.compile("(@)([a-zA-Z0-9.-]+)")
         userTags = pattern.findall(note)
         if userTags:
-            RunningNotesDataHandler.notify_tagged_user(application, userTags, project, note, category, user, timestamp)
+            RunningNotesDataHandler.notify_tagged_user(application, userTags, proj_ids, note, category, user, timestamp)
```

Comment:
Wouldn't it be easier to just add a new argument with the project name?

Reply:
I didn't want to make the argument list any longer 😅